### PR TITLE
Enh/configurable gateset

### PR DESCRIPTION
**Pull request by AdriaanRol**
Makes the gateset of the lutman configurable.

### CODE REVIEW

**Comment by AdriaanRol** [comment]
@slavoutich , I forgot why we didn't merge this yet.

File: pycqed/measurement/openql_experiments/generate_qi_cfg.py [hunk 1]
```diff
@@ -54,13 +54,13 @@ def _rotation_instruction(phi, theta):
```
Comment (slavoutich) [inline pycqed/measurement/openql_experiments/generate_qi_cfg.py]: Shouldn't generate config files be removed at all already now?

File: pycqed/measurement/openql_experiments/openql_helpers.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 output_dir = join(dirname(__file__), 'output')
 ql.set_option('output_dir', output_dir)
```
Comment (slavoutich) [inline pycqed/measurement/openql_experiments/openql_helpers.py]: Helpers are not extremely needed anymore, IMO.

File: pycqed/measurement/openql_experiments/generate_qi_cfg.py (hunk 1 above)
Comment (AdriaanRol) [reply pycqed/measurement/openql_experiments/generate_qi_cfg.py]: No, we can remove this only once QINF is properly integrated.

File: pycqed/measurement/openql_experiments/openql_helpers.py (hunk 2 above)
Comment (AdriaanRol) [reply pycqed/measurement/openql_experiments/openql_helpers.py]: Same as above.

**Comment by AdriaanRol** [comment]
This was closed manually.

**Pull request closed without merging**